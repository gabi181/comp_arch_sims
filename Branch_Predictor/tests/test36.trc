32 1 8 0 global_history local_tables not_using_share
0x598220E0 N 0x6449708C
0x598220E0 T 0x6449708C
0x6807B068 T 0x0F8A31FC
0x6807B068 N 0x0F8A31FC
0x6807B068 N 0x0F8A31FC
0x598220E0 N 0x6449708C
