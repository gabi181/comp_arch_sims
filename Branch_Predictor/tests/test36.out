0x598220e0 N 0x598220e4
0x598220e0 N 0x598220e4
0x6807b068 N 0x6807b06c
0x6807b068 N 0x6807b06c
0x6807b068 N 0x6807b06c
0x598220e0 N 0x598220e4
flush_num: 2, br_num: 6, size: 1377b
